tests/unit/test_aioshumway.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import json
+
+import pytest
+
+import aioshumway
+
+
+@pytest.fixture
+def transport(mocker):
+    return mocker.Mock(asyncio.transports.DatagramTransport)
+
+
+class TestFFWDClientProtocol:
```

Comment:
These test classes come from when I first began thinking about pytest + test classes. Curious to know what you think!